source/ota.c

```diff
@@ -305,6 +305,8 @@ static OtaStateTableEntry_t otaTransitionTable[] =
     { OtaAgentStateAll,                 OtaAgentEventShutdown,            shutdownHandler,        OtaAgentStateStopped             },
 };
 
+
+
```

Comment:
Nit: These extra lines can be removed

Reply:
Done: https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/112/commits/130a2cafd84f8880cf198de5ac2fe2542fdcd831